pawn/Elements/AdministratorCommands.pwn

```diff
@@ -972,6 +972,11 @@ lvp_show(playerId, params[]) {
         showInfo = true;
         format(g_message, sizeof(g_message), "Save money and stats! Register your nickname at www.sa-mp.nl");
     }
+    
+    else if (!strcmp(showParameter, "discord", true, 4)) {
```

Comment:
The "4" in this command needs to be a "7", as it reflects the length of the string that has to be matched.

`/show discovery` will now also show this message, which isn't intentional.